R/learner_aorsf_surv_aorsf.R

```diff
@@ -7,6 +7,20 @@
 #' Calls [aorsf::orsf()] from \CRANpkg{aorsf}.
 #' Note that although the learner has the property `"missing"` and it can in principle deal with missing values,
 #' the behaviour has to be configured using the parameter `na_action`.
+#'
+#'
+#' @details
+#' Five types of prediction are allowed for this learner:
```

Comment:
maybe mention that these are not mlr3 predict types, at least I was confused initially

Reply:
Oh yes, it's alright, I will write it